src/Command/InfectionCommand.php

```diff
@@ -164,8 +164,8 @@ protected function initialize(InputInterface $input, OutputInterface $output)
             }
         }
 
-        if (!defined('HHVM_VERSION') && !extension_loaded('xdebug')) {
-            throw new \Exception('You need to install and enable xDebug in order to allow for code coverage generation.');
+        if (PHP_SAPI !== 'phpdbg' && !defined('HHVM_VERSION') && !extension_loaded('xdebug')) {
```

Comment:
Please also add a check to `app/infection.php`, line 5: because now there is a warning 

```php
Warning: Infection may only be invoked from a command line
```

It should be `if (PHP_SAPI !== 'cli' && PHP_SAPI !== 'phpdbg') {`

Reply:
done